DEPLOYMENT_TASKS.md

```
@@ -82,6 +82,82 @@ Keep an item unchecked until the action has actually been performed and verified
+
+The Firestore rule change accepts the optional millisecond timestamp already
+written by notification actions. Deploy it before releasing the client so
+existing action-created RSVP documents can be changed in older and newer apps.
+
+- [ ] Deploy the RSVP-compatible Firestore rules:
+
+  ```sh
+  npx firebase deploy --project prod --only firestore:rules
+  ```
+
+      Verify a signed-in non-admin can change an existing Interested RSVP that
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)